src/components/Footer/index.tsx

```diff
@@ -165,11 +166,21 @@ export const Footer: FunctionComponent<FooterPropsType> = props => {
           </FooterSectionLinks>
 
           <FooterSection>
-            <Text textSize="s">
-              Cancer Research UK is a registered charity in England and Wales (1089464), Scotland (SC041666), the Isle
-              of Man (1103) and Jersey (247). Registered as a company limited by guarantee in England and Wales No.
-              4325234.
-            </Text>
+            {props.theme?.name === 'su2c' ? (
```

Comment:
Could we make the footer text a theme value like `theme.siteConfig.footerText` rather than using the theme name here?

Reply:
this was something @PeteLindsell  suggested but I wasn't sure if we should large blocks of text in the theme

Reply:
I agree that if it needs to changed in the future it'd probably be better for it to remain this way. Happy to discuss this

Reply:
Hmm, as of now though it's just one paragraph of text. If we were going to make it more complicated than that we would probably revisit this component anyway.

I'd prefer that anything we have that varies _with_ the theme becomes part _of_ the theme. We could do something like `theme.siteConfig.footerText = 'cruk' or 'su2c'` and then use that in here instead of using the theme name inside components?

I've written similar patterns to this before, it gets confusing to remember places the theme name has been used like this. What would happen if we added a new theme called `su2c-v2` that was based on the su2c theme? We'd have to change the condition in every place we did this to `props.theme?.name === 'su2c' || props.theme?.name === 'su2c-v2'`

Reply:
That is still less text than copying the same footer text into multiple themes, I have no strong feeling about this I happy either way.

Reply:
I can make the change @dgcollard